### PR TITLE
Add Freebase QA Dataset

### PR DESCRIPTION
**Pull request by gchhablani**
Closes PR #1435. Fixed issues with PR #1809.

Requesting @lhoestq to review.

### CODE REVIEW

File: datasets/freebase_qa/README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+languages:
+- en
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- extended|trivia-qa
```
Comment (lhoestq) [inline datasets/freebase_qa/README.md:15]:
```suggestion
- extended|trivia_qa
```

File: datasets/freebase_qa/README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    month = jun,
+    year = "2019",
+    address = "Minneapolis, Minnesota",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/N19-1028",
+    doi = "10.18653/v1/N19-1028",
+    pages = "318--323",
+    abstract = "In this paper, we present a new data set, named FreebaseQA, for open-domain factoid question answering (QA) tasks over structured knowledge bases, like Freebase. The data set is generated by matching trivia-type question-answer pairs with subject-predicate-object triples in Freebase. For each collected question-answer pair, we first tag all entities in each question and search for relevant predicates that bridge a tagged entity with the answer in Freebase. Finally, human annotation is used to remove any false positive in these matched triples. Using this method, we are able to efficiently generate over 54K matches from about 28K unique questions with minimal cost. Our analysis shows that this data set is suitable for model training in factoid QA tasks beyond simpler questions since FreebaseQA provides more linguistically sophisticated questions than other existing data sets.",
+}
+```
```
Comment (lhoestq) [inline datasets/freebase_qa/README.md:165]:
````suggestion
```
### Contributions

Thanks to [@gchhablani](https://github.com/gchhablani) for adding this dataset.
````
We have a new section now ;)

File: datasets/freebase_qa/README.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
```
Comment (lhoestq) [inline datasets/freebase_qa/README.md:45]:
```suggestion
  - [Citation Information](#citation-information)
  - [Contributions](#contributions)
```

**Review by lhoestq — APPROVED** [review]
Nice ! thank you :)

**Comment by gchhablani** [comment]
Hi @lhoestq 

Thanks for approving. Request you to close PR #1435 as well.

**Pull request merged**